packages/config/test/__snapshots__/options.test.js.snap

```diff
@@ -305,6 +305,7 @@ Object {
     "host": "localhost",
     "https": false,
     "port": 3000,
+    "rateLimit": undefined,
```

Comment:
We should have a full configuration for ratelimt in snapshot

Reply:
It is initially set `undefined` by default :)

Reply:
I mean we may need to have all configurable keys tested, maybe in middleware unit test

Reply:
Agree @clarkdo. Will include them for unit test.